.travis.yml

```diff
@@ -42,7 +42,7 @@ before_script:
   - cd web && npm install && BROWSERIFYSWAP_ENV='development' npm run bundleProduction && cd ..
 
 script:
-  - "mvn -Dkey=$API_KEY clean test verify -B"
+  - "mvn -Dkey=$API_KEY clean test verify install -B"
```

Comment:
this seems to have fixed the build ? should we keep it like this ?

Reply:
Yes, the install forces maven to always fetch the dep from local repo. A bit strange as I thought that maven knows this already, but maybe also a bug of that forbiddenapis plugin. A bit strange that it passes for openjdk8...

Reply:
Ok, `install` should be enough (instead of `verify install`) since install already includes verify, Ill do this in master and then merge this.

Reply:
And thanks a lot for figuring this one out!